index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>tribute page</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+
```

Comment:
Good job with using inline comments feature 👍  But it is not pointing to a specific line or element. Unfortunately, I have no idea which element you are referring to. The other thing is you need to explain why you are requesting this change. Otherwise, it would be your personal opinion. 👍